pimd/pim_igmp.h

```diff
@@ -56,17 +56,23 @@
 #define IGMP_V3_GROUP_RECORD_SOURCE_OFFSET     (8)
 #define IGMP_CHECKSUM_OFFSET                   (2)
 
-/* RFC 3376: 8.1. Robustness Variable - Default: 2 */
-#define IGMP_DEFAULT_ROBUSTNESS_VARIABLE           (2)
+/* RFC 3376: 8.1. Robustness Variable - Default: 2 for IGMP */
+/* RFC 2710: 7.1. Robustness Variable - Default: 2 for MLD */
+#define GM_DEFAULT_ROBUSTNESS_VARIABLE           (2)
```

Comment:
Shouldn't these go into somewhere else common for IGMP/MLD? As they are now under IGMP :/

Reply:
Ping

Reply:
Yes Donatos, will be moving it to the common function.